### PR TITLE
Python client tutorial

### PR DESCRIPTION
**Pull request by dalton-boggs**

### CODE REVIEW

File: 3_python_client/.env [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export STAC_FASTAPI_TITLE=stac-fastapi-lucenia
```
Comment (arichardet) [inline 3_python_client/.env]: looks like some of this file is from tutorial 2

File: 3_python_client/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
+# Running STAC Catalog Demo
```
Comment (arichardet) [inline 3_python_client/README.md]: from STAC demo tutorial 2

File: 1_getting-started/node/config/lucenia.yml [hunk 3]
```diff
@@ -1,3 +1,26 @@
 cluster.name: "lucenia-cluster"
 network.host: 0.0.0.0
 plugins.license.certificate_filepath: config/trial.crt
+
```
Comment (arichardet) [inline 1_getting-started/node/config/lucenia.yml:4]: this file was an accidental commit?

**Review by arichardet — COMMENTED** [review]
Nice. Following the blog draft, I got:

```
Total flood zones: 1092
```

File: 1_getting-started/node/config/lucenia.yml (hunk 3 above)
Comment (nknize) [reply 1_getting-started/node/config/lucenia.yml:4]: @dalton-boggs looks like this PR hasn't been merged yet? Can you remove the following so we can merge?

Comment (dalton-boggs) [reply 1_getting-started/node/config/lucenia.yml:4]: Yes, merging now.

**Pull request merged**